package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/v4-localization",
-  "version": "1.1.46",
+  "version": "1.1.47",
```

Comment:
hm should we add back the deleted blurb from [here](https://github.com/dydxprotocol/v4-localization/pull/326/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L3)? or do you think it's probably ok 👀

Reply:
looks like probably ok I think

Reply:
not totally sure what it is for either